--- libowfat-0.34-orig/GNUmakefile	2024-10-31 14:42:37.000000000 +0100
+++ libowfat-0.34/GNUmakefile	2026-03-15 12:48:28.363548753 +0100
@@ -4,10 +4,10 @@
 # in /opt/diet, where they are in the default search path for my diet libc
 # work but don't conflict with anything there.  YMMV.
 
-prefix=/opt/diet
+prefix=/usr
 LIBDIR=${prefix}/lib
 INCLUDEDIR=${prefix}/include
-MAN3DIR=${prefix}/man/man3
+MAN3DIR=${prefix}/share/man/man3
 
 LIBS=byte.a fmt.a scan.a str.a uint.a open.a stralloc.a unix.a socket.a \
 buffer.a mmap.a taia.a tai.a dns.a case.a mult.a array.a io.a \
--- libowfat-0.34-orig/Makefile	2024-10-31 14:42:49.000000000 +0100
+++ libowfat-0.34/Makefile	2026-03-15 12:48:28.364187601 +0100
@@ -5,10 +5,10 @@
 # in /opt/diet, where they are in the default search path for my diet libc
 # work but don't conflict with anything there.  YMMV.
 
-prefix=/opt/diet
+prefix=/usr
 LIBDIR=${prefix}/lib
 INCLUDEDIR=${prefix}/include
-MAN3DIR=${prefix}/man/man3
+MAN3DIR=${prefix}/share/man/man3
 
 LIBS=byte.a fmt.a scan.a str.a uint.a open.a stralloc.a unix.a socket.a \
 buffer.a mmap.a taia.a tai.a dns.a case.a mult.a array.a io.a \
